admin/src/views/users/components/search/index.js

```diff
@@ -8,7 +8,7 @@ import { withApollo } from 'react-apollo';
 import { withRouter } from 'react-router';
 import { Spinner } from '../../../../components/globals';
 import { throttle } from '../../../../helpers/utils';
-import { searchUsersQuery } from 'shared/graphql/queries/search/searchUsers';
+import { searchUsersQuery } from '@spectrum/shared/graphql/queries/search/searchUsers';
```

Comment:
I don't think Flow supports this kind of aliases

Reply:
Yes it does via module.name_mapper

Reply:
Heck they should be using typescript anyways! :D 

Shots fired!!!